<ui version="4.0" >
 <class>QDelimiterUi</class>
 <widget class="QDialog" name="QDelimiterUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="3" column="0" >
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>&amp;Size:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>sizeCO</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sizeCO" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" >
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QListWidget" name="leftLW" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="iconSize" >
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="resizeMode" >
         <enum>QListView::Adjust</enum>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="currentRow" >
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="rightLW" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="iconSize" >
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="resizeMode" >
         <enum>QListView::Adjust</enum>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" >
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>50</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="insertPB" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Insert the delimiters</string>
        </property>
        <property name="text" >
         <string>&amp;Insert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePB" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="texCodeL" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>3</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>TeX Code: </string>
      </property>
      <property name="textFormat" >
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QCheckBox" name="matchCB" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>3</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip" >
       <string>Match delimiter types</string>
      </property>
      <property name="text" >
       <string>&amp;Keep matched</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>matchCB</tabstop>
  <tabstop>sizeCO</tabstop>
  <tabstop>insertPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_helpers.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
